sdk/src/locking/fast_async_lock.rs

```diff
@@ -0,0 +1,38 @@
+use crate::locking::IggySharedMutFn;
+use fast_async_mutex::rwlock::{RwLock as FastAsyncRwLock, RwLockReadGuard, RwLockWriteGuard};
+use std::sync::Arc;
+
+#[cfg(feature = "fast_async_lock")]
```

Comment:
it's not needed in this file, since this file is always included if feature `fast_async_lock` is enabled